app/models/response.rb

```diff
@@ -160,7 +160,7 @@ def substitute_variables(obj)
       mentor_title: mentor&.role&.title,
       mentor_gender: mentor&.gender,
       mentor_name: mentor&.first_name,
-      organization: student.role.organization.name,
+      organization: student.role.team.organization.name,
```

Comment:
De waarde vervangt in de vragenlijst de variabele `je_begeleidingsinitiatief`. Klopt dat dat dat hier de organization name moet zijn? Of moet het nou juist de team name zijn?

Reply:
ja voor zover ik weet klopt het zo. In het ticket hadden we iig dit staan:

```
3. je_begeleidingsinitiatief moet gaan naar de nieuwe organisatie

(Zowel in vragenlijst als in sms tekst)
```

https://freedcamp.com/Compsy_oK1/VSV_wNL/todos/15278431/